app/models/event.rb

```diff
@@ -1,9 +1,61 @@
-class Event < ActiveRecord::Base
+class Event
+  attr_reader :url, :title, :description, :venue_name, :venue_address,
+              :all_day, :latitude, :longitude, :geocode_type, :start_time,
+              :end_time, :venue_city, :image, :categories
+
+  def initialize(data)
+    image           = data["image"] ? data["image"]["medium"]["url"] : "default.jpg"
+    @url            = data["url"],
+    @title          = data["title"],
+    @description    = data["description"],
+    @venue_name     = data["venue_name"],
+    @venue_address  = data["venue_address"],
+    @all_day        = set_all_day_boolean(data["all_day"]),
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Unnecessary spacing detected.